src/js/videojs.record.js

```diff
@@ -535,9 +535,12 @@ class Record extends Plugin {
                         // join microphone track with screencast stream (order matters)
                         screenStream.addTrack(mic.getTracks()[0]);
                         this.onDeviceReady.bind(this)(screenStream);
-                    }).catch(
-                        this.onDeviceError.bind(this)
-                    );
+                    }).catch((code) => {
+                        if (screenStream.active) {
```

Comment:
add a comment why this check is added, e.g. permissions error. more info is better.